UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/UserService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
```

Comment:
We have implemented UserService but not used it yet, I think we will use it in upcoming PBI related to redirect distributors for tracings info.

Reply:
Yes, we will use it upcoming PBI to verify if user is distributor based on specified role